src/panel.js

```diff
@@ -100,13 +103,43 @@ const addFullWidthButton = (fragment, listenerClass) => {
   return button;
 };
 
+const addTooltip = (wrapper, stringId) => {
+  const div = document.createElement("div");
+  div["id"] = stringId;
+  setClassAndAppend(wrapper, div);
+};
+
 
 const addSpan = (wrapper, stringId) => {
   const span = document.createElement("span");
   span["id"] = stringId;
   setClassAndAppend(wrapper, span);
 };
 
+const getActiveHostname = async() => {
+  const tabsQueryResult = await browser.tabs.query({currentWindow: true, active: true});
+  const currentActiveTab = tabsQueryResult[0];
+  const currentActiveURL = new URL(currentActiveTab.url);
+  const thisHostname = currentActiveURL.hostname;
+  return thisHostname;
+};
+
+const isSiteInContainer = async(panelId) => {
+
+  if (panelId === "on-facebook") {
+    // Site is on default FBC domain. Show the "remove site" button, in a disabled state.
+    return true;
+  }
+
+  const addedSitesList = await browser.runtime.sendMessage("what-sites-are-added");
+  const hostname = await getActiveHostname();
+
+  if (addedSitesList.includes(hostname)) {
+    return true;
+  } else {
```

Comment:
nit: drop the `else` - `return true` will exit the function so the `else` block is not needed. I trim a lot of `else` blocks out of my code just for readability.